openspec/changes/archive/2026-04-17-slash-command-transport-parity/exploration.md

```
@@ -0,0 +1,52 @@
+## Exploration: issue #541 slash command transport parity
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)